src/main/java/org/sonatype/repository/conan/internal/proxy/ConanProxyFacet.java

```diff
@@ -130,33 +140,46 @@ private Content putPackage(final String assetPath,
   private Content putMetadata(final String assetPath,
                               final Content content,
                               final AssetKind assetKind,
+                              final String group,
                               final String project,
-                              final String version,
-                              final String group)
+                              final String version)
       throws IOException
   {
     StorageFacet storageFacet = facet(StorageFacet.class);
     try (TempBlob tempBlob = storageFacet.createTempBlob(content.openInputStream(), HASH_ALGORITHMS)) {
       AttributesMap attributesMap;
-      TempBlob blob = tempBlob;
       switch (assetKind) {
         case DOWNLOAD_URL:
-          blob = absoluteUrlRemover.removeAbsoluteUrls(tempBlob, getRepository());
-          // break intentionally left out!
```

Comment:
Is there a reason you dropped this comment, and if so should this have a break?

Reply:
The reason it intentionally dropped through was so that ConanManifest.parse would also be called. This is now handled entirely in the case statement. There is no need for a break statement because we now return from the case statement.

Reply:
Yup duh. 😄  The try threw me off the scent. 👍